runtime/queries/column_timeseries.go

```diff
@@ -103,24 +104,30 @@ func (q *ColumnTimeseries) Resolve(ctx context.Context, rt *runtime.Runtime, ins
 		}
 
 		measures := normaliseMeasures(q.Measures, q.Pixels != 0)
-		dateTruncSpecifier := convertToDateTruncSpecifier(timeRange.Interval)
+		timeBucketSpecifier := convertToTimeBucketSpecifier(timeRange.Interval)
 		tsAlias := tempName("_ts_")
 		temporaryTableName := tempName("_timeseries_")
+
+		timezone := "UTC"
+		if q.TimeZoneAdjustment != nil && q.TimeZoneAdjustment.GetTimeZone() != "" {
+			timezone = q.TimeZoneAdjustment.GetTimeZone()
+		}
 		querySQL := `CREATE TEMPORARY TABLE ` + temporaryTableName + ` AS (
 			-- generate a time series column that has the intended range
 			WITH template as (
 			SELECT
-				range as ` + tsAlias + `
+				timezone('` + timezone + `',range) as ` + tsAlias + `
```

Comment:
This is a SQL injection opportunity – can we pass it as an arg and use `?` instead?

Reply:
Yes, I thought about it, the question signs makes SQL quite unreadable. On the other hand, security is of higher priority, cannot argue here.